js/coinex.js

```diff
@@ -1471,9 +1472,14 @@ module.exports = class coinex extends Exchange {
         const swap = market['swap'];
         const stopPrice = this.safeString2 (params, 'stopPrice', 'stop_price');
         const postOnly = this.safeValue (params, 'postOnly', false);
-        const reduceOnly = this.safeValue (params, 'reduceOnly');
         const positionId = this.safeInteger2 (params, 'position_id', 'positionId'); // Required for closing swap positions
         let timeInForce = this.safeString (params, 'timeInForce'); // Spot: IOC, FOK, PO, GTC, ... NORMAL (default), MAKER_ONLY
+        const reduceOnly = this.safeValue (params, 'reduceOnly');
+        if (reduceOnly !== undefined) {
+            if (market['type'] !== 'swap') {
+                throw new InvalidOrder (this.id + ' createOrder() does not support reduceOnly for ' + market['type'] + ' orders, reduceOnly orders are supported for swap markets only');
+            }
+        }
```

Comment:
We should omit it from `params`, right? Also, it looks like adding this param doesn't actually do anything?

Reply:
reduceOnly is already omitted and there's already functionality for swap orders, if reduceOnly is set to true it uses the swap close order endpoints.